src/Main.java

```diff
@@ -0,0 +1,26 @@
+/**Implement  builder design pattern by taking Person with the attributes FirstName, LastName, age, address … upto 10 attributes
+Is the same result can be achieved by using setters? 
+If yes, demonstrate
+If no, demonstrate
+Is your code mutable?  
+If yes, demonstrate 
+If no, demonstrate */
+public class Main {
+    public static void main(String[] args) {
+
+        Person person1 = new PersonBuilder("Vamshi",23,"address","male","11/05/1999").buildMutablePerson();
+        System.out.println(person1);
+        //Yes, the same result can be achieved by using setters
+        Person person2 = new PersonBuilder()
+                            .setName("Vamshi").setAddress("address").setAge(23).setDob("11/05/1999").setGender("male").setMaritalStatus("single").buildMutablePerson();
```

Comment:
The problem statement is talking about the setters of the original Person class. Not the setters from PersonBuilder class.